contracts/nft/src/query/validate.rs

```diff
@@ -0,0 +1,28 @@
+use cosmwasm_std::{Coin, Deps};
+
+use crate::core::error::ContractError;
+use crate::{core::msg::QueryMsg, util::validate::Validate};
+
+impl Validate for QueryMsg {
+    fn validate(&self, _deps: Deps) -> Result<(), ContractError> {
+        match self {
+            QueryMsg::AllNftInfo { .. } => Ok(()),
```

Comment:
You had some validation within the query handlers. You can probably move it into here if you still plan to keep this.

Reply:
alright should be fixed now